src/Symfony/Component/HttpKernel/Controller/ControllerResolverInterface.php

```diff
@@ -42,16 +42,4 @@
      * @throws \LogicException If the controller can't be found
      */
     public function getController(Request $request);
-
-    /**
-     * Returns the arguments to pass to the controller.
-     *
-     * @param Request  $request    A Request instance
-     * @param callable $controller A PHP callable
-     *
-     * @return array An array of arguments to pass to the controller
-     *
-     * @throws \RuntimeException When value for argument given is not provided
-     */
-    public function getArguments(Request $request, $controller);
```

Comment:
We won't remove a method from an interface, that's a BC break and we never allowed that to happen in a minor version. But, we did deprecate methods on interfaces before, we just always found a way to make it work in all cases. I haven't had a look at how this could be possible, but I will later on.